packages/backend/bindings/node/index.ts

```diff
@@ -89,6 +89,8 @@ export function init(id: string, storagePath?: string): { destroy: () => void; r
     _poll(
         runtime,
         (error, data) => {
+            // Swallow error thrown when poll returns after the actor is destroyed
+            if (error && error.message.includes("receiving on a closed channel")) return
```

Comment:
```suggestion
            if (error && error.message.includes('receiving on a closed channel')) return
```

`prettier` is complaining about the double quotes